src/components/dateTimePicker/index.js

```diff
@@ -167,7 +183,7 @@ class DateTimePicker extends BaseComponent {
 
   renderDateTimePicker() {
     const {chosenDate, showExpandableOverlay} = this.state;
-    const {mode, minimumDate, maximumDate} = this.props;
+    const {mode, minimumDate, maximumDate, locale, is24Hour, minuteInterval, timeZoneOffsetInMinutes} = this.props;
```

Comment:
I guess at this point we can pass on the other props as `...other`
```suggestion
    const {mode, minimumDate, maximumDate, ...others} = this.props;
```